_source/_posts/2020-02-03-managing-multiple-okta-instances-with-terraform-cloud.md

```diff
@@ -165,85 +179,90 @@ Another nice collaboration feature of Terraform Cloud is that you can leave comm
 
 ## Manage Multiple Okta Environments with Terraform
 
-Now that you're using Terraform Cloud to manage a single environment (production), let's add a second environment (development), which will make it easy for your developers to run tests against this new Okta environment without impacting production.
+Now that you're using Terraform Cloud to manage a single environment (production). Let's add a second environment (development), which will make it easy for your developers to run tests against this new Okta environment without impacting production.
 
 If you don't have a second Okta org yet, [go create one](https://developer.okta.com/signup/), and then go through the steps to create an API token once more.
 
-Next, create a new Git branch in your project called `dev`. This is where you'll work on your product before releasing it to production (the `master` branch).
+Next, create a new Git branch in your project called `dev`. This is where you'll work on your product before releasing it to production (the `main` branch).
 
 ```console
 git checkout -b dev
 git push origin dev
 ```
 
-In Terraform Cloud, create a new workspace to represent this new environment. Target the same repository as before but set the branch specifier to `dev`. For this environment, set the apply method in Terraform Cloud to `auto` so changes will be applied to the environment immediately if the plan stage is successful without you needing to manually confirm anything.
+In Terraform Cloud, create a new workspace to represent this new environment. Target the same repository as before but set the branch specifier to `dev`.
 
-{% img blog/okta-terraform-cloud/terraform-cloud-create-workspace.gif alt:"Terraform Cloud create workspace" width:"800" %}{: .center-image }
+{% img blog/okta-terraform-cloud/terraform-cloud-create-workspace.jpg alt:"Terraform Cloud create workspace" width:"800" %}{: .center-image }
 
-This time, set up new variables for your workspace using your *new* Okta org's credentials. Click **Queue plan** and just like that, Terraform replicated your production Okta configuration from your first Okta org into the second Okta org!
+For this environment, set the apply method in Terraform Cloud to `auto` so changes will be applied to the environment immediately if the planning stage is successful without you needing to confirm anything manually.
+
+{% img blog/okta-terraform-cloud/terraform-cloud-create-workspace-2.jpg alt:"Terraform Cloud workspace settings" width:"800" %}{: .center-image }
+
+Update the `okta.auto.tfvar` file using your _new_ Okta org's `org_name` and `base_url` and create a new variable for `api_token` from this org. Click **Start new plan** and just like that, Terraform replicated your production Okta configuration from your first Okta org into the second Okta org!
```

Comment:
```suggestion
Update the `okta.auto.tfvar` file using your _new_ Okta org's `org_name` and `base_url` and create a new variable for `api_token` for this org. Click **Start new plan** and just like that, Terraform replicated your production Okta configuration from your first Okta org into the second Okta org!
```

Reply:
Is this correct? should it be "for"?

Reply:
Not exactly but I see that the sentence is not clear, I'll rephrase